runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/RedistributeByKeyOverrideFactory.java

```diff
@@ -135,6 +135,7 @@ public Duration getAllowedTimestampSkew() {
                 public void processElement(
                     @Element KV<K, ValueInSingleWindow<V>> kv,
                     OutputReceiver<KV<K, V>> outputReceiver) {
+                  // todo #33176 specify additional metadata in the future
```

Comment:
Yea I think we'll want to find a way to update Redistribute to automatically carry along "all" metadata. We built a separate data type for Redistribute to avoid making WindowedValue public. But now that it is public and everything around it is public (or public enough) we should make it possible to be resilient to new metadata being added without having to edit Redistribute or the overrides.